object_tracking_with_gcp/Object Tracking Tutorial.ipynb

```diff
@@ -0,0 +1,381 @@
+    "\n",
+    "annotations_bucket = gcp_client.bucket(ANNOTATIONS_BUCKET)\n",
+    "\n",
+    "if not annotations_bucket.exists():\n",
+    "    gcp_client.create_bucket(ANNOTATIONS_BUCKET)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Upload videos to the bucket\n",
+    "videos = Path(\"./data/videos/\").glob(\"*\")\n",
+    "for video in tqdm(videos):\n",
+    "    blob = bucket.blob(video.name)\n",
+    "    blob.upload_from_filename(str(video))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 46,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "45it [00:25,  1.76it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Upload videos to the bucket\n",
+    "videos = Path(\"./data/output_annotations/\").glob(\"*\")\n",
+    "for annotation in tqdm(videos):\n",
+    "    blob = annotations_bucket.blob(annotation.name)\n",
+    "    blob.upload_from_filename(str(annotation))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create your AIPlatform dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from google.cloud import aiplatform"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Generate links to csv files\n",
+    "gcs_source = [f\"gs://{ANNOTATIONS_BUCKET}/{p.name}\" for p in Path(\"./data/output_annotations/\").glob('*')]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 55,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_dataset(\n",
+    "    project: str,\n",
+    "    location: str,\n",
+    "    display_name: str,\n",
+    "    sync: bool,\n",
+    "    gcs_source: list\n",
+    "):\n",
+    "    aiplatform.init(project=project, location=location)\n",
+    "\n",
+    "    ds = aiplatform.VideoDataset.create(\n",
+    "        display_name=display_name,\n",
+    "        import_schema_uri=aiplatform.schema.dataset.ioformat.video.object_tracking,\n",
+    "        gcs_source = \"gs://object_tracking_bucket/\",\n",
+    "        sync=sync\n",
+    "    )\n",
+    "\n",
+    "    ds.wait()\n",
+    "\n",
+    "    print(ds.display_name)\n",
+    "    print(ds.resource_name)\n",
+    "    return ds\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DS = create_dataset(\"test_from_jupyter\", LOCATION, \"test_from_jupyter\", True, gcs_source)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running training and inference"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Converting predictions into SuperAnnotate format"
```

Comment:
Can you show predictions inGCP?

Reply:
Not yet, once the training is complete sure, but it will be still images of videos. Can't add GIFs.